src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier color = new ColorSupplier();
+
+    public String getRandomBall() {
```

Comment:
why do you limit yourself with this return type? You should return a regular Ball object, it's up to the client/user to transform it to the String if it's needed, but if someone needed a whole object, he would need to parse it from String :)